lib/Service/Sync/ImapToDbSynchronizer.php

```diff
@@ -122,7 +122,11 @@ public function syncAccount(Account $account,
 		int $criteria = Horde_Imap_Client::SYNC_NEWMSGSUIDS | Horde_Imap_Client::SYNC_FLAGSUIDS | Horde_Imap_Client::SYNC_VANISHEDUIDS): void {
 		$rebuildThreads = false;
 		foreach ($this->mailboxMapper->findAll($account) as $mailbox) {
-			if (!$mailbox->isInbox() && !$mailbox->getSyncInBackground()) {
+			$trashMailboxId = $account->getMailAccount()->getTrashMailboxId();
+			$trashRetentionDays = $account->getMailAccount()->getTrashRetentionDays();
```

Comment:
Could it work to move $trashMailboxId and $trashRetentionDays before the loop?

Reply:
Yeah sure. Done.